packages/fxa-settings/src/styles/ctas.scss

```diff
@@ -48,7 +48,7 @@
   }
 
   &:focus {
-    @apply border-0;
+    @apply border-transparent;
```

Comment:
Awesome! While we're here, since this button has almost no visual indication when it's focussed, I think we should add a dotted outline on focus, similar to the rest of the buttons. The other buttons' focus outlines work right out of the box because it inherits the text color (black). In the red button's case the text is white, so the focus outline _is_ actually there, just not visible. Let's make this focus outline black:

In the [Tailwind config](https://github.com/mozilla/fxa/blob/main/packages/fxa-react/configs/tailwind.js), under the `extend` property extend the `outline` classes with:

```js
outline: {
  'black-dotted': '1px dotted #000',
},
```

Then you can `@apply` this new class to the `:focus` ruleste of `cta-caution`:

```
@apply border-transparent outline-black-dotted;
```

You may need to `yarn build-postcss` in `fxa-settings` to make these styles take effect, but this should give you a focus outline on the red buttons.